go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/dell/terraform-provider-redfish
 
-go 1.18
+go 1.19
```

Comment:
is it possible to support 1.20?

Reply:
The gofish library that we are using is at golang 1.16 and Terraform SDK framework is at 1.19.  so it was discussed that once we will move from sdk framework to plugin framework in future then we will move to 1.20 version of go